src/Decoder.ts

```diff
@@ -20,6 +20,17 @@ export type DecoderOptions<ContextType = undefined> = Readonly<
      */
     useBigInt64: boolean;
 
+    /**
+     * By default, string values will be decoded as UTF-8 strings. However, if this option is true,
+     * string values will be returned as Uint8Arrays without additional decoding.
+     * 
+     * This is useful if the strings may contain invalid UTF-8 sequences.
+     * 
+     * Note that this option only applies to string values, not map keys. Additionally, when
+     * enabled, raw string length is limited by the maxBinLength option.
+     */
```

Comment:
Why no do this for map keys as well?

Reply:
This library doesn't support binary keys, so I decided to skip that case.